sklearn/README.md

```diff
@@ -0,0 +1,154 @@
+
+## How to create conda environment for benchmarking
+
+If you want to test stock sklearn, then use  
+
+`conda create -n skl_bench -c conda-forge python=3.7 scikit-learn pandas`
+
+If you want to test scikit-learn in the Intel(R) Distribution for Python*, then use 
+
+`conda create -n skl_bench -c intel python=3.7 scikit-learn pandas`
+
+and set the appropriate environment variable
+
+`export USE_DAAL4PY_SKLEARN=YES`
+
+##  Algorithms parameters
+
+Benchmarks can be launched for each algorithm separately. The list of supported parameters for each algorithm provided below.
```

Comment:
```suggestion
You can launch benchmarks for each algorithm separately. The tables below list all supported parameters for each algorithm:

- [General](#general)
- [DBSCAN](#dbscan)
- [RandomForestClassifier](#randomforestclassifier)
- [RandomForestRegressor](#randomforestregressor)
- [pairwise_distances](#pairwise_distances)
- [KMeans](#kmeans)
- [KNeighborsClassifier](#kneighborsclassifier)
- [LinearRegression](#linearregression)
- [LogisticRegression](#logisticregression)
- [PCA](#pca)
- [Ridge Regression](#ridge)
- [SVC](#svc)
- [train_test_split](#train_test_split)
```

sidenote: rename the "General" section, e.g. "Common parameter" or something like this